### PR TITLE
Ccvweb bt bg video

### PR DESCRIPTION
**Pull request by btweedy2879**
Add background video to home page header.

### CODE REVIEW

File: RockWeb/Themes/church_ccv_External_v8/Styles/pages/home/home.css [hunk 1]
```diff
@@ -4,6 +4,62 @@
     background-size: unset;
 }
 
+#bg-video-wrapper {
+    /*height: 50vw;
+    min-height: 50vh;*/
```
Comment (cupshir) [inline RockWeb/Themes/church_ccv_External_v8/Styles/pages/home/home.css]: Should this be removed?

File: RockWeb/Themes/church_ccv_External_v8/Styles/pages/home/home.css [hunk 2]
```diff
@@ -4,6 +4,62 @@
...
+    position: relative;
+}
+
+    #bg-video-wrapper .bg-video {
```
Comment (cupshir) [inline RockWeb/Themes/church_ccv_External_v8/Styles/pages/home/home.css]: is indent intentional?

File: RockWeb/Themes/church_ccv_External_v8/Styles/pages/home/home.css [hunk 3]
```diff
@@ -4,6 +4,62 @@
...
+        margin: none;
+        position: inherit;
+        z-index: 1;
+        top: 50%;
+        left: 0%;
+        transform: translate(0%, -50%);
+        height: auto;
+        width: 100%;
+    }
+
```
Comment (cupshir) [inline RockWeb/Themes/church_ccv_External_v8/Styles/pages/home/home.css]: extra space

File: RockWeb/Themes/church_ccv_External_v8/Styles/pages/home/home.css (hunk 2 above)
Comment (btweedy2879) [reply RockWeb/Themes/church_ccv_External_v8/Styles/pages/home/home.css]: No.  It was added by VS when I saved.   Must be a formatting setting in the rock vs config settings?

Comment (btweedy2879) [reply RockWeb/Themes/church_ccv_External_v8/Styles/pages/home/home.css]: But went ahead and removed it.

**Pull request merged**